thunderfish/sortfishtype.py

```diff
@@ -0,0 +1,189 @@
+import numpy as np
```

Comment:
I am not happy with the name of the module. But it is hard to find a better one. Algorithm wise there is a bias towards pulse fishes, so why not naming it maybe pulsefishdetection ?Better ides? 

And: Tills powerspectrum method should also go into this module.